app/models/manageiq/providers/ibm_power_hmc/infra_manager/provision.rb

```diff
@@ -0,0 +1,7 @@
+class ManageIQ::Providers::IbmPowerHmc::InfraManager::Provision < MiqProvision
+  include_concern 'Cloning'
+  include_concern 'StateMachine'
+  def destination_type
+    "Vm"
```

Comment:
If you're provisioning _to_ a template then the destination_type should be `"Template"

Reply:
You should also validate that the request_type is `"clone_to_template"`, see https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/infra_manager/provision.rb#L9 for examples